proxy/api/src/ethereum_address.rs

```diff
@@ -0,0 +1,35 @@
+use serde::{Deserialize, Serialize};
```

Comment:
I would prefer if we would group these `ethereum_` modules in an `ethereum` directory. So we would have

`ethereum/`
  - `address.rs`
  - `claim_ext.rs`

Reply:
Fixed